Fixed errors in QuantityKinds vocabulary and added new IEC units into…

jhodgesatmb opened the pull request:
… the unit vocabulary (ongoing)

steveraysteveray commented on vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:
```diff
@@ -3962,6 +4242,66 @@ unit:DecaC
 .
+unit:DecaGM
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1.0e-01 ;
```
Should be 0.01?

jhodgesatmb replied:
fixed

steveraysteveray commented on vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:
```diff
@@ -4051,6 +4391,42 @@ unit:DeciC
 .
+unit:DeciGM
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e-01 ;
```
Should be 1e-4

jhodgesatmb replied:
fixed

steveraysteveray commented on vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:
```diff
@@ -4051,6 +4391,42 @@ unit:DeciC
+.
+unit:DeciL
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e-01 ;
```
0.1 Liters, but 1e-4 m**3, so should be 1e-4.

jhodgesatmb replied:
fixed

steveraysteveray commented on vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:
```diff
@@ -4069,6 +4445,126 @@ unit:DeciM
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e-02 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "0,01-fold of the power of the SI base unit metre with the exponent 2" ;
```
Would be "0,1-fold..."  if the description is trying to say "one tenth of a Meter, squared"
Or, it is saying "one-hundredth of a square meter", but that isn't a good way to think of it, because the "Deci" applies to the M, and then the result is squared.

jhodgesatmb replied:
fixed

steveraysteveray commented on vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:
```diff
@@ -4069,6 +4445,126 @@ unit:DeciM
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e-03 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "0,001-fold of the power of the SI base unit metre with the exponent 3" ;
```
Similar to the DeciM2 unit, I would say "One tenth of a meter, cubed"
Same logic would apply to all the DeciM... units.

jhodgesatmb replied:
fixed

steveraysteveray commented on vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:
```diff
@@ -4069,6 +4445,126 @@ unit:DeciM
+  a qudt:Unit ;
+  qudt:conversionMultiplier 9.071847e+01 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "100-fold of the SI base unit kilogram" ;
```
This seems an odd description, since this is not equal to 100 kilograms.

jhodgesatmb replied:
fixed

steveraysteveray commented on vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:
```diff
@@ -4480,6 +5024,18 @@ unit:ExaC
+  qudt:conversionMultiplier 1e+18 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "1 000 000 000 000 000 000-fold of the derived SI unit joule" ;
+  qudt:hasQuantityKind quantitykind:ThermalEnergy ;
```
Just "Energy"

jhodgesatmb replied:
fixed

steveraysteveray commented on vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:
```diff
@@ -1443,18 +1443,10 @@ unit:BTU_IT-PER-HR-FT2-DEG_R
-  <http://qudt.org/schema/hasQuantityKind> quantitykind:SpecificGibbsEnergy ;
-  <http://qudt.org/schema/hasQuantityKind> quantitykind:SpecificHelmholtzEnergy ;
-  <http://qudt.org/schema/hasQuantityKind> quantitykind:SpecificInternalEnergy ;
   qudt:conversionMultiplier 0e+00 ;
```
Should not be zero.

jhodgesatmb replied:
fixed

steveraysteveray commented on vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl:
```diff
@@ -1480,15 +1467,10 @@ unit:BTU_IT-PER-LB_F-DEG_F
-  <http://qudt.org/schema/hasQuantityKind> quantitykind:SpecificHeatCapacityAtConstantPressure ;
-  <http://qudt.org/schema/hasQuantityKind> quantitykind:SpecificHeatCapacityAtConstantVolume ;
-  <http://qudt.org/schema/hasQuantityKind> quantitykind:SpecificHeatCapacityAtSaturation ;
   qudt:conversionMultiplier 0e+00 ;
```
Should not be zero

jhodgesatmb replied:
fixed

steveraysteveray replied:
Still looks like it is zero.

jhodgesatmb commented:
I guess I fixed a different one. I have fixed unit:BTU_IT-PER-LB_F-DEG_R CM
to: 4.269e+02

I have changed the CM for BBL_US_DRY based on the fact that dry barrel is
26.25 US Gallons -> 1.156281989625e-01

On Fri, Mar 27, 2020 at 11:11 AM steveraysteveray <notifications@github.com>
wrote:

> *@steveraysteveray* requested changes on this pull request.
>
> While you are fixing the one remaining comment, would you mind also fixing
> the zero CM for:
> unit:BBL_US_DRY
>
> Somehow, it snuck by both of us before.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/qudt/qudt-public-repo/pull/73#pullrequestreview-383093714>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AATQRWOERRL4SZGUJWRSJNTRJTT43ANCNFSM4LUTVPPA>
> .
>


-- 
Jack
